about/src/main/java/dev/arkbuilders/components/about/About.kt

```diff
@@ -0,0 +1,303 @@
+                    }
+                }
+                HorizontalDivider(
+                    modifier = Modifier.padding(top = 20.dp),
+                    thickness = 1.dp,
+                    color = ArkColor.BorderSecondary
+                )
+                Row(modifier = Modifier.padding(top = 12.dp, bottom = 50.dp)) {
+                    OutlinedButton(
+                        modifier = Modifier,
+                        onClick = { ctx.openLink(ctx.getString(R.string.ark_contribute_url)) },
+                        border = BorderStroke(
+                            width = 1.dp,
+                            color = ArkColor.BorderSecondary
+                        ),
+                        shape = RoundedCornerShape(8.dp),
+                        contentPadding = PaddingValues(0.dp)
+                    ) {
+                        Text(
+                            modifier = Modifier.padding(8.dp),
+                            text = stringResource(R.string.about_discover_issues_to_work_on),
+                            color = ArkColor.TextSecondary,
+                            fontSize = 12.sp,
+                            fontWeight = FontWeight.Medium
+                        )
+                    }
+                    OutlinedButton(
+                        modifier = Modifier.padding(start = 12.dp),
+                        onClick = { },
+                        border = BorderStroke(
+                            width = 1.dp,
+                            color = ArkColor.BorderSecondary
+                        ),
+                        shape = RoundedCornerShape(8.dp),
+                        contentPadding = PaddingValues(0.dp),
+                        enabled = false
+                    ) {
+                        Text(
+                            modifier = Modifier.padding(8.dp),
+                            text = stringResource(R.string.about_see_open_bounties),
+                            color = ArkColor.TextPlaceHolder,
+                            fontSize = 12.sp,
+                            fontWeight = FontWeight.Medium
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun SocialLink(painter: Painter, text: String, onClick: () -> Unit) {
+    TextButton(
+        modifier = Modifier.defaultMinSize(
+            minWidth = 1.dp,
+            minHeight = 1.dp
+        ),
+        contentPadding = PaddingValues(horizontal = 4.dp, vertical = 4.dp),
+        onClick = { onClick() },
+        shape = RoundedCornerShape(4.dp)
+    ) {
+        Icon(
+            modifier = Modifier.size(20.dp),
+            painter = painter,
+            contentDescription = text,
+            tint = Color.Unspecified
+        )
+        Text(
+            modifier = Modifier.padding(start = 4.dp),
+            text = text,
+            color = ArkColor.TextTertiary
+        )
+    }
+}
```

Comment:
@mdrlzy Should we move this Composable to separate class to enhance readability? The class starts becoming big.